odata-json-format/19 Batch Requests and Responses.md

```diff
@@ -383,11 +393,9 @@ value is a string containing the header-encoded value of the header.
 
 The response object MAY contain the name/value pair `body`
 which follows the same rules as within [request objects](#BatchRequest).
-
-If the media type is not exactly equal to `application/json`
-(i.e. it is a subtype or has format parameters), the
-`headers` object MUST contain a name/value pair with the name
-`content-type` whose value is the media type.
+The `content-type` header mandated by [#OData-Protocol#HeaderContentType]
+MAY be omitted if the media type is exactly equal to `application/json`
+(i.e. it is not a subtype and has no format parameters).
```

Comment:
Is the same exception also made for the `content-type` header in JSON batch _requests_?